plugins/ptp_operator/event/event.go

```diff
@@ -135,22 +176,24 @@ func (p *PTPEventState) UpdateCurrentEventState(c ClockState) ptp.SyncState {
 				"node": clockState.NodeName, "iface": alias}).Set(float64(v))
 		}
 		clockState.Metric = metrics
-		p.DependsOn[c.Process] = clockState
+		p.DependsOn[c.Process] = []*ClockState{clockState}
 	}
 
 	// if all locked then its locked
 	// if anyone HOLDOVER then holdover
 	// if  anyone FREERUN then freerun
```

Comment:
comments not clear. What the currentState should be if DependsOn states are a mix of HOLDVER and FREERUN?

Reply:
if anyone is HOLDOVER then holdover comment 2